governance/pyth_staking_sdk/src/utils/clock.ts

```diff
@@ -0,0 +1,17 @@
+import { BN } from "@coral-xyz/anchor";
+import { Connection } from "@solana/web3.js";
+import { EPOCH_DURATION } from "../constants";
+
+export const getCurrentSolanaTimestamp = async (connection: Connection) => {
+  const slot = await connection.getSlot();
+  const blockTime = await connection.getBlockTime(slot);
+  return new BN(blockTime!);
```

Comment:
Same comment about avoiding `!`, also should we be returning bigint here instead of BN to be consistent with other APIs?

Reply:
fixed.